src/blocks/forms/edit.js

```diff
@@ -334,9 +334,10 @@ const UAGBFormsEdit = ( props ) => {
 };
 
 const addAdvancedClasses = createHigherOrderComponent( ( BlockListBlock ) => {
-	return ( props ) => {
-		return <BlockListBlock { ...props } className={ props.attributes.className } />;
-	};
+	return ( props ) => (
+		// First we add any existing classes, then we add our classes if needed.
+		<BlockListBlock { ...props } className={ `${ props?.className ? `${ props.className } ` : '' }${ props?.attributes?.className ? props.attributes.className : '' }` } />
```

Comment:
```suggestion
	const addClassNames = uagbClassNames( [
			props?.className || '',
			props?.attributes?.className || ''
		] );

		<BlockListBlock { ...props } className={ addClassNames ? addClassNames : null } />
```
For clean code we can keep like this

Reply:
Updated the same.